datumaro/plugins/splitter.py

```diff
@@ -227,24 +228,33 @@ def _split_dataset(self):
 
 class ReidentificationSplit(_TaskSpecificSplit):
     """
-    Splits dataset for re-identification.|n
+    Splits a dataset for re-identification task.|n
+    Produces a split with a specified ratio of images, avoiding having same
+    labels in different subsets.|n
```

Comment:
Here, we avoid having the same `person id` or `object id`.  It could be `label` or `attribute` if `attr_for_id` is specified.

Reply:
One more thing is, actually `train` and `val` set shares `person id` or `object id`. (Most person re-identification data doesn't have `val` set though).  But they do not share `IDs` with `test` set.
I'm not sure how accurate the explanation should be.
If you feel the current explanation is sufficient, please leave it as it is.